apps/framework-cli/src/framework/core/infrastructure_map.rs

```diff
@@ -2038,7 +2057,7 @@ impl PartialInfrastructureMap {
 
             let data_model = crate::framework::data_model::model::DataModel {
                 name: partial_api.name.clone(),
-                // TODO pass through version from the TS / PY api
+                // TODO: what is this version?
                 version: Version::from_string("0.0".to_string()),
```

Comment:
ApiEndpoint has a data model object in it. What should this version? Same as the api?

Reply:
I wonder if that should be a datamodel or a reference. ie the datamodel Id. that way we would not have issues

Reply:
I think we'll drop the `DataModel` from the infra map when we no longer have dmv1?